Source/Core/Common/ChunkFile.h

```diff
@@ -235,7 +235,7 @@ class PointerWrap
 
 		while (true)
 		{
-			u8 shouldExist = (list_cur ? 1 : 0);
+			u8 shouldExist = !!list_cur;
```

Comment:
Because bool is not an u8? And I don't think true/false are required to evaluate to 0/1 respectively (albeit they usually do).
What part of the ternary expression didn't you like?